go/libkb/test_common.go

```diff
@@ -276,7 +276,7 @@ func SetupTest(tb testing.TB, name string, depth int) (tc TestContext) {
 		pc, file, line, ok := runtime.Caller(depth)
 		if ok {
 			fn := runtime.FuncForPC(pc)
-			fmt.Printf("SetupTest %s %v:%v\n", fn.Name(), file, line)
+			fmt.Fprintf(os.Stderr, "- SetupTest %s %s:%d\n", filepath.Basename(fn.Name()), filepath.Base(file), line)
```

Comment:
It works in ci. File path is like path but works on Windows.

For the second questions because the function names have file paths in them

On Tuesday, July 12, 2016, Patrick Crosby <notifications@github.com> wrote:

> In go/libkb/test_common.go
> <https://github.com/keybase/client/pull/3425#discussion_r70445589>:
>
> > @@ -276,7 +276,7 @@ func SetupTest(tb testing.TB, name string, depth int) (tc TestContext) {
> >  		pc, file, line, ok := runtime.Caller(depth)
> >  		if ok {
> >  			fn := runtime.FuncForPC(pc)
> > -			fmt.Printf("SetupTest %s %v:%v\n", fn.Name(), file, line)
> > +			fmt.Fprintf(os.Stderr, "- SetupTest %s %s:%d\n", filepath.Basename(fn.Name()), filepath.Base(file), line)
>
> And why call filepath anything on a function name?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3425/files/315c2bd221a64e0df746d26712f59362d37e5ea2#r70445589>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA05_3YkgYf4_98D0Ut5t84s48ZuDFKOks5qU6BSgaJpZM4JKXPL>
> .
>